metriq_gym/benchmarks/quantum_volume.py

```diff
@@ -20,7 +22,7 @@ def dispatch_handler(self) -> None:
             stats = calc_stats([result], self.params["confidence_level"])
             logging.info(f"Simulator-only job completed: {stats[0]}.")
             print(stats[0])
-            return
+            # return
```

Comment:
leftover comment

Reply:
I actually think this `return` should simply be retained. We don't want to put simulator-only jobs in the queue file for polling: they don't correspond to job IDs on IBMQ.

Reply:
Ah, yes, for sure; somehow it looked like a void return at the end of the method

Reply:
The `return` statement has `return`ed.